### PR TITLE
Add more Views to Jenkins UI

### PR DESCRIPTION
**Pull request by BruceForstall**
Add specific views for:
1. All push jobs (view named "Merge")
2. All periodic jobs
3. A view per architecture
4. A view per OS

This allows looking at a much smaller subset of the entire set of jobs.

None of the specific views include PR jobs.

### CODE REVIEW

**Comment by BruceForstall** [comment]
@mmitche @RussKeldorph @jashook PTAL

Is it safe to ask for a CI test? This doesn't change any jobs, just views.

**Comment by mmitche** [comment]
@BruceForstall Yeah

File: netci.groovy [hunk 1]
```diff
@@ -950,15 +1072,15 @@ def static addNonPRTriggers(def job, def branch, def isPR, def architecture, def
 // flow job that rolls up the build and test for non-windows OS's.  // If the job is a windows build only job,
 // it's just used for internal builds
 // If you add a job with a trigger phrase, please add that phrase to coreclr/Documentation/project-docs/ci-trigger-phrases.md
-def static addTriggers(def job, def branch, def isPR, def architecture, def os, def configuration, def scenario, def isFlowJob, def isWindowsBuildOnlyJob) {
+def static addTriggers(def jobName, def job, def branch, def isPR, def architecture, def os, def configuration, def scenario, def isFlowJob, def isWindowsBuildOnlyJob) {
```
Comment (mmitche) [inline netci.groovy]: I think you can get away without passing jobName here.  You should be able to access the job;s name with job.name inside addTriggers.

Comment (BruceForstall) [reply netci.groovy]: Nice; that simplifies things a lot.

**Comment by BruceForstall** [comment]
@dotnet-bot test CI please

**Comment by BruceForstall** [comment]
@dotnet-bot test CI please

**Comment by BruceForstall** [comment]
@dotnet-bot test CI please

**Comment by BruceForstall** [comment]
@dotnet-bot test CI please

**Comment by BruceForstall** [comment]
@mmitche @jashook @RussKeldorph This looks pretty good to me. Check out the new views at the bottom of https://ci.dot.net/job/dotnet_coreclr/job/master/job/GenPRTest/job/generator_prtest/738/.

Only thing I'm not sure of is it looks like maybe the "Official Builds" view disappeared? Not sure where that is created for the root. We still have "Official Builds" for jitstress and illink folders.

Comments?

**Comment by BruceForstall** [comment]
@dotnet-bot test CI please

**Comment by BruceForstall** [comment]
@mmitche @RussKeldorph @jashook ping

**Pull request merged**